rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -1566,9 +1557,6 @@ static dds_qos_t * create_readwrite_qos(
     case RMW_QOS_POLICY_LIVELINESS_UNKNOWN:
       dds_qset_liveliness(qos, DDS_LIVELINESS_AUTOMATIC, ldur);
       break;
-    case RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE:
-      dds_qset_liveliness(qos, DDS_LIVELINESS_MANUAL_BY_PARTICIPANT, ldur);
-      break;
     case RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_TOPIC:
       dds_qset_liveliness(qos, DDS_LIVELINESS_MANUAL_BY_TOPIC, ldur);
       break;
```

Comment:
Does this need a default case statement?

Reply:
Done https://github.com/ros2/rmw_cyclonedds/pull/178/commits/7f8a45167a74a12f27572e9ba101a8e30d29ea8c.